docs/languages/typescript.md

```diff
@@ -255,48 +255,17 @@ The  `editor.codeActionsOnSave` setting lets you configure a set of Code Actions
     "source.organizeImports": true,
 }
 ```
-You can also enable or disable which Code Actions are run on save per language using a language specific setting:
-```json
-"[typescript]": {
-    "editor.codeActionsOnSave": {
-        "source.organizeImports": true
-    }
-}
-```
+
 Here are some source actions:
 
 * `"organizeImports"` -  Enables organize imports on save.
 * `"fixAll"` - Auto Fix on Save computes all possible fixes in one round (for all providers including ESLint).
 * `"fixAll.eslint"` -  Auto Fix only for ESLint.
 * `"addMissingImports"` - Adds all missing imports on save.
 
-The TSLint extension already uses `source.fixAll` Code Action kind to implement fix all and auto fix on save.
-
-You can also set `editor.codeActionsOnSave` to an array of Code Actions to execute in order. You can use this to guarantee that a specific Code Action is always run before or after another one that may conflict with it.
-The following `editor.codeActionsOnSave` will always run Organize Imports followed by Fix All once organize imports finishes:
-```json
-"editor.codeActionsOnSave": [
-    "source.organizeImports",
-    "source.fixAll"
-]
-```
-Note: `editor.codeActionsOnSave` supports both object and array configs in settings.
+See [TypeScript](/docs/typescript/typescript-tutorial) for more information.
```

Comment:
I don't know if you prefer to link to a subsection instead but I found the subsection links just in case:

https://code.visualstudio.com/docs/getstarted/settings#_language-specific-editor-settings

For organizeImports specifically in TypeScript:
https://code.visualstudio.com/docs/languages/typescript#_organize-imports